tests/Feature/ProductTest.php

```diff
@@ -2,54 +2,44 @@
 
 use App\Models\Product;
 
-
 it('user can add a new product', function () {
 
     userLogin();
 
     $product = Product::factory()->create();
 
-    $this->post(route('product.store'), [
+    $productData = [
         'name' => 'PDU',
         'brand_id' => $product->brand_id,
         'size_id' => $product->size_id,
         'color_id' => $product->color_id,
-        'status' => '0',
-    ])
+        'status' => true,
+    ];
+
+    $this->post(route('product.store'), $productData)
         ->assertStatus(302)
         ->assertRedirect(route('product.index'));
 
-    $this->assertDatabaseHas('products', [
-        'name' => 'PDU',
-        'brand_id' => $product->brand_id,
-        'size_id' => $product->size_id,
-        'color_id' => $product->color_id,
-        'status' => '0',
-    ]);
+    $this->assertDatabaseHas('products', $productData);
 });
 
 it('user can update a product', function () {
 
     userLogin();
 
     $product = Product::factory()->create();
```

Comment:
product factory ma static value seed karo ane pachi value ene update karvani try karo jethi update that che khyal ave.